woocommerce/storefront
# Add .nvmrc file with version 12.21.0 specified

Fixes https://github.com/woocommerce/storefront/issues/1811

### Description
With the latest version of Node the npm run build command has some issues creating the required minified versions of required JS files and produces a broken build. This occurred during a deployment, you can read more on this incident report pca54o-2gc-p2

### Solution
To mitigate this issue from happening I've added a `.nvmrc` file specifying a compatible version of Node (12.21.0)

### How to test the changes in this Pull Request:

1. Install [nvm](https://github.com/nvm-sh/nvm). Skip this step if you already have it installed.
2. Using terminal navigate to your local `storefront` repository root
3. Check that upon navigating into the repo that nvm installs Node 12.21.0
4. Double check the correct version by running `node -v`

### Changelog

> Add – .nvmrc to ensure developers are running a compatible version of node. #1811
